static/styles/message_row.css

```diff
@@ -1,7 +1,7 @@
 $avatar_column_width: 46px;
```

Comment:
Most of the CSS changes can only easily be explained as "I tried to match the same height of a one-line post with a profile picture and a one-line post without", while trying to make everything look good.

Reply:
Update: now with 3px extra of height.